cmd/kogito/command/shared/resource_checks_test.go

```diff
@@ -16,7 +16,7 @@ package shared
 
 import (
 	"github.com/kiegroup/kogito-cloud-operator/api/v1beta1"
-	"github.com/kiegroup/kogito-cloud-operator/pkg/test"
+	test2 "github.com/kiegroup/kogito-cloud-operator/core/test"
```

Comment:
Nitpick but the name can be removed and call the functions directly as `test.` since I don't see any other conflicting package

Reply:
done